### ISSUE TITLE
Kräva API-nyckel vid skapande av restaurang

### ISSUE THREAD

**Issue by jwilsson** [jh222vp/Webb-API#5]
Bara min personliga åsikt, man kanske inte ska kunna skapa en ny restaurang utan en API-nyckel. En JWT-token krävs ju men kanske även en API-nyckel?

**Comment by 1dv450** [comment]
Jo, API-nyckeln bör finnas med vid VARJE anrop till APIet. Oftast är den ju till för statistik.
